src/server/handlers/manifest-handler.ts

```diff
@@ -9,15 +9,15 @@ let manifest: Awaited<ReturnType<typeof generateManifestJson>> | undefined =
   undefined;
 
 export default async (req: Request, res: Response) => {
-  if (!manifest || manifest.start_url !== getHttpBaseExternal()) {
```

Comment:
`start_url` is hard-coded to `/`, so this if statement was always true with the right hand side of the or expression. Removing that condition should hopefully mean the manifest gets generated if it's not already stored in memory.